04/index.html

```diff
@@ -1,30 +1,44 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/83859d448f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./styles/global.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(min-width: 768px)" href="./styles/desktop.css">
 </head>
+
 <body>
-    
+
     <nav>
-        <ul>
-            <li>
+        <label for="menu-switcher">
+            <i class="fas fa-bars"></i>
+        </label>
+        <input type="checkbox" id="menu-switcher">
+        <ul class="menu-list">
+            <li class="menu-list-item">
                 <a href="#">start</a>
             </li>
-            <li>
+            <li class="menu-list-item">
                 <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
+                <ul class="submenu-list">
+                    <li class="menu-list-item submenu-list-item"><a href="">strony internetowe</a></li>
+                    <li class="menu-list-item submenu-list-item"><a href="">pozycjonowanie stron www</a></li>
                 </ul>
             </li>
-            <li>
+            <li class="menu-list-item">
```

Comment:
Jeśli używasz BEM to trochę dziwne, że wszystkie elementy to bloki - jednak widać tu ewidentnie też elementy :)

PS. Samo menu wygląda bardzo dobrze :)